### PR TITLE
fix exit cmd issue on Mac

### PR DESCRIPTION
**Pull request by zacklocx**
I'm a Mac user, I found that the postExitCmd config didn't work on Mac, so I reviewed the code and made this commit to fix this.

### CODE REVIEW

**Comment by sacredbanana** [comment]
Nice. Have you checked to see if it also performs nicely if you don't have any post exit commands set?

**Comment by zacklocx** [comment]
> Nice. Have you checked to see if it also performs nicely if you don't have any post exit commands set?

I made some more detail improve with some new commits, as I tested it performed OK if there is no post exit commands set when app exit

File: src/Misc.h [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 namespace Misc
 {
+  extern const char * postExitCmd;
```
Comment (Gargaj) [inline src/Misc.h]: I'd much rather not have this, it looks gross.

Comment (zacklocx) [reply src/Misc.h]: yeah, I just made the code simple and keep consistent with other code style already in this project, like here:

namespace Renderer
{
  extern const char * defaultShaderFilename;

but indeed I can do it better by offering an interface such as setter/getter

Comment (Gargaj) [reply src/Misc.h]: The `ExecuteCommand` function itself can internally test if the incoming path is NULL or empty string, that's probably the cleanest way.

Comment (zacklocx) [reply src/Misc.h]: we need to know the exit cmd script path in other place, for Mac, when tap "Quit" on TouchBar, we need to know where is the script path

so in TouchBar.mm, I made this change which require an access of postExitCmd

- (void)quit {
     Misc::ExecuteCommand(Misc::defaultPostExitCmd, Renderer::defaultShaderFilename);
     exit(0);
 }

Comment (zacklocx) [reply src/Misc.h]: to make code clean, I just commit this:

[make code clean](https://github.com/Gargaj/Bonzomatic/pull/105/commits/0af97504dabc2d65ae9f80bbbb1cdbe5db5cec55)

File: src/Renderer.h [hunk 2]
```diff
@@ -25,6 +25,9 @@ namespace Renderer
   bool OpenSetupDialog( RENDERER_SETTINGS * settings );
   bool Open( RENDERER_SETTINGS * settings );
   
+  void ShowCursor();
```
Comment (Gargaj) [inline src/Renderer.h]: These seem like a different commit

File: src/main.cpp [hunk 3]
```diff
@@ -59,6 +59,8 @@ int main(int argc, const char *argv[])
     options.parse( szConfig );
   }
 
+  bool bShowMode = false;
```
Comment (Gargaj) [inline src/main.cpp]: So is this.

**Comment by Gargaj** [comment]
Fixed in #107

**Pull request closed without merging**